plugins/outputs/sql/sql_test.go

```diff
@@ -201,6 +201,8 @@ func TestMysqlIntegration(t *testing.T) {
 	p.Driver = "mysql"
 	p.DataSourceName = address
 	p.InitSQL = "SET sql_mode='ANSI_QUOTES';"
+	err = p.Init()
+	require.NoError(t, err)
```

Comment:
```suggestion
	require.NoError(t, p.Init())
```
Same for all occurrences.